client/src/helpers.js

```diff
@@ -1,19 +1,53 @@
+import React from 'react';
```

Comment:
I love that you created a helper functions file for your front-end. Just curious, why isn't this a .jsx file?

Reply:
I don't think it would work as a .jsx file since I would have to wrap it in {} and I'm not using any JSX in that file. It's just functions.